src/app/shared/event-log/event-log-table.html

```diff
@@ -12,7 +12,9 @@
            [target]="'_blank'"
            *ngIf="event.state_id !== 'Draft'">{{event.event_name}}</a><span
       *ngIf="event.state_id === 'Draft'">{{event.event_name}}</span></td>
-    <td>{{event.timestamp | ccdDate: 'short'}}</td>
+    <td><div class="tooltip">{{event.timestamp | ccdDate : 'local'}}
```

Comment:
Why are we changing this?

Reply:
@MKaramath added a local date modifier and a tooltip for the event details component, however this table seems to have been forgotten. Which leads different time displayed in log table and detail. This is just for consistency.

Reply:
👍